deploy/examples/object-openshift.yaml

```diff
@@ -135,9 +135,26 @@ spec:
 apiVersion: route.openshift.io/v1
 kind: Route
 metadata:
-  name: rook-ceph-rgw-my-store # We recommend name to be the same as the service name below, but it is not required
+  name: rook-ceph-rgw-my-store-http
   namespace: rook-ceph # namespace:cluster
 spec:
+  port:
+    targetPort: http
   to:
     kind: Service
     name: rook-ceph-rgw-my-store # The name of the RGW service is in the form 'rook-ceph-rgw-<objectstore-name>'
+#---
+#apiVersion: route.openshift.io/v1
+#kind: Route #if TLS is enabled
+#metadata:
+#  name: rook-ceph-rgw-my-store-https
+#  namespace: rook-ceph # namespace:cluster
+#spec:
+#  port:
+#    targetPort: https
+#  tls:
+#    insecureEdgeTerminationPolicy: Redirect
+#    termination: reencrypt
+#  to:
+#    kind: Service
+#    name: rook-ceph-rgw-my-store # The name of the RGW service is in the form 'rook-ceph-rgw-<objectstore-name>'
```

Comment:
I wonder if it would make sense to have this in a separate YAML file instead of the comment.

Reply:
@subhamkrai any opinion? The route is only supported in openshift, so IMHO it make to sense include in `object-openshift.yaml`

Reply:
Instead of a separate route, how about just adding comments to the existing route CR above so it's easy to just uncomment ~4 lines to enable https.